Assets/Scripts/Platform/PlatformColor.cs

```diff
@@ -27,36 +28,37 @@ private void Update()
 
             if (PlayerProperties.playerColor.Equals(platformColor) && !PlayerProperties.playerColor.Equals(playerColor))
             {
-                collider.isTrigger = false;
+                StartCoroutine(WaitForX(10f));
             }
             else
             {
-                collider.isTrigger = true;
+                collider.enabled = false;
             }
         }
     }
 
-    private void OnCollisionEnter2D(Collision2D collision)
+    IEnumerator WaitForX(float waitTime)
     {
-        CheckCollision(collision.collider);
+        Debug.Log("startedwait");
+        yield return new WaitForSeconds(waitTime);
+        collider.enabled = true;
```

Comment:
Toto presuň za metódu v na riadok 32. Nech sa v metóde na čakanie len čaká a nič iné.